src/Symfony/Component/Workflow/MarkingStore/ScalarMarkingStore.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\MarkingStore;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * ScalarMarkingStore.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ScalarMarkingStore implements MarkingStoreInterface, UniqueTransitionOutputInterface
+{
+    private $property;
+    private $propertyAccessor;
+
+    public function __construct($property = 'marking')
+    {
+        $this->property = $property;
+        $this->propertyAccessor = PropertyAccess::createPropertyAccessor();
```

Comment:
Seems like you have updated `PropertyAccessorMarkingStore` but not the `ScalarMarkingStore` ;)